samples/RNWinRTTestApp/App.tsx

```diff
@@ -159,17 +160,35 @@ function AsyncImage(props: AsyncImageProps) {
 const App = () => {
     updateJumpListAsync();
     // NOTE: The Id used is the hash of the string 'SampleProvider'. This Guid is 'eff1e128-4903-5093-096a-bdc29b38456f'
-    const loggingChannel = new Windows.Foundation.Diagnostics.LoggingChannel("SampleProvider", null);
+    const loggingChannel = new Windows.Foundation.Diagnostics.LoggingChannel("SampleProvider", null!);

```

Comment:
what is `null!` about? 

I don't think I've ever seen this before?

Reply:
Although the function does accept null as a valid parameter, TS tooling were doing strict checks and threw errors in these lines. This was in VS Code.

By adding a postfix !, I'm telling TS that it can ignore null/undefined checks for this variable, and pass the null.